internal/grpc_dialer.go

```diff
@@ -53,8 +61,13 @@ const (
 )
 
 func dial(params connectionParameters) (*grpc.ClientConn, error) {
```

Comment:
I believe it should be rather `dialParameters` or `connect()`.

Reply:
Dial was its original name and whole file is named ```grpc_dialer```